### DISCUSSION TITLE
Report Headers Enums

### DISCUSSION THREAD

**Discussion by dehidehidehi** [saleweaver/python-amazon-sp-api#81] [Ideas]
I have found myself creating Enums for some report headers.
This is very useful when using DictReader from the csv module for report file reading.

So far, I've just done two: 
GET_MERCHANT_LISTINGS_ALL_DATA 
GET_MERCHANT_LISTINGS_DATA 

Do you think it's something that is pull worthy?

**Comment by saleweaver** [comment]
Yes absolutely. You can find all report values in the repo's ReportType Enum.
